ZelBack/src/services/explorerService.js

```diff
@@ -450,6 +450,29 @@ async function processStandard(blockDataVerbose, database) {
       }
     }
   }));
+  if (appsTransactions.length > 0) {
+    const options = {
+      ordered: false, // If false, continue with remaining inserts when one fails.
+    };
+    await dbHelper.insertManyToDatabase(database, appsHashesCollection, appsTransactions, options);
+    if (blockDataVerbose.height >= config.fluxapps.fluxAppRequestV2) {
+      while (appsTransactions.length > 500) {
+        appsService.checkAndRequestMultipleApps(appsTransactions.splice(0, 500));
+        // eslint-disable-next-line no-await-in-loop
+        await serviceHelper.delay(60 + (Math.random() * 14) * 1000); // delay 60 and 75 seconds
```

Comment:
I think that the processing now will be actualy slower than before. As after every block there might be some significant delay.

Reply:
Have reduced the delay in case we are already on fluxAppRequestV2 to a max of 10 seconds between each 500 messages... getting 500 app messages on a block, hopefully will hapen soon but I don't see it slowing down the block processing any time soon :)
I have removed the await before the fluxAppRequestV2 as currently we also do not await in there.